__tests__/app.test.js

```diff
@@ -180,3 +181,28 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/reviews", () => {
+  test("200: Returns an array of review objects, sorted by descending date order", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((response) => {
+        const reviewsArr = response.body.reviews;
```

Comment:
Nice refactor here could be to use the key you're sending back 
eg `const {reviews} = response.body`
Makes it slightly cleaner code, and you wouldn't be naming the data type in your variable name (pro tip - don't do this if you can avoid it 😄 )